packages/ivh-base-style/dist/ivh-base-style.css

```diff
@@ -1575,6 +1562,33 @@ body::-webkit-scrollbar-track {
   grid-column-end: end;
 }
 
+.au-table-slim th,
+.au-table-slim td {
+  padding: 7px 10px;
+}
+
+.au-table-relaxed th,
+.au-table-relaxed td {
+  padding: 13px 10px;
+}
+
+.au-table-borderless {
+  border-top:none;
+}
+
+.au-table-borderless th:first-child,
+.au-table-borderless td:first-child,
+.au-table-borderless > *:first-child,
+.au-table-borderless th,
+.au-table-borderless td,
+.au-table-borderless > * {
+  border: unset !important;
```

Comment:
Is the `!important` required here?

Reply:
important isn't necessary if you include thead th

Reply:
I dislike !important so I will remove it.